cms/documents/migrations/0004_alter_customdocument__privacy.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.1.3 on 2024-11-29 16:20
```

Comment:
any chance of merging the migrations?

Reply:
Not really. Any existing documents and images are added with an initial `_privacy` value of 'public', because it's likely already referenced by pages, and it's too late to hook into 'page_published' signals to change that.

The second migration for each app changes the default value to 'private', so that newly added documents and images receive that status.

Reply:
I was looking at the volume of images/docs on dev/staging and it super low (so the question made sense to me 🙈 ), but I agree this is the proper way when there is content.